etc/helm/pachyderm/templates/pachd/rbac/worker-role.yaml

```diff
@@ -16,6 +16,7 @@ rules:
   - ""
   resources:
   - services
+  - pods
```

Comment:
this is something I'm just noticing because we're adding a resource here, but I don't think the worker should be able to modify any k8s resources. I can open another PR removing the modify verbs here

Reply:
yep makes sense!